lightning/src/ln/functional_tests.rs

```diff
@@ -7738,11 +7738,11 @@ fn test_bump_penalty_txn_on_revoked_htlcs() {
 	let payee = Payee::from_node_id(nodes[1].node.get_our_node_id()).with_features(InvoiceFeatures::known());
 	let scorer = test_utils::TestScorer::with_fixed_penalty(0);
 	let route = get_route(&nodes[0].node.get_our_node_id(), &payee, &nodes[0].network_graph, None,
-		3_000_000, 50, nodes[0].logger, &scorer).unwrap();
+		3_000_000, 50, 1000,  nodes[0].logger, &scorer).unwrap();
```

Comment:
nit: remove extra space

Reply:
Done in https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/b7cb0f3eca8f491de09c97e41b8f090c3d27264e.